src/core/mod.rs

```diff
@@ -307,6 +307,24 @@ impl BPF {
         }
     }
 
+    pub fn get_syscall_prefix(&mut self) -> Result<String, Error> {
+        for prefix in SYSCALL_PREFIXES.iter() {
+            if self.ksymname(prefix).is_ok() {
+                return Ok((*prefix).to_string());
```

Comment:
I don't think you need to write this like this, should be able to: `return Ok(prefix.to_string());`

Reply:
I followed Clippy on this one
```terminal
warning: calling `to_string` on `&&str`
   --> src/core/mod.rs:323:27
    |
323 |                 return Ok(prefix.to_string());
    |                           ^^^^^^^^^^^^^^^^^^ help: try dereferencing the receiver: `(*prefix).to_string()`
    |
    = note: `#[warn(clippy::inefficient_to_string)]` on by default
    = help: `&str` implements `ToString` through a slower blanket impl, but `str` has a fast specialization of `ToString`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#inefficient_to_string
```

Reply:
TIL. Resolving this one